scripts/release/Dockerfile

```
@@ -105,31 +105,24 @@ FROM alpine:latest
 # libc6-compat
 RUN apk add --no-cache libstdc++ binutils
 
-# Create the user/group for stump
-RUN addgroup -g 1000 stump
-RUN adduser -D -s /bin/sh -u 1000 -G stump stump
-
 WORKDIR /
 
 # create the config, data and app directories
 RUN mkdir -p config && \
     mkdir -p data && \
     mkdir -p app
 
-# FIXME: this does not seem to be working...
-# make the stump user own the directories
-RUN chown stump /config && \
-    chown stump /data && \
-    chown stump /app
-
-USER stump
-
 # copy the binary
-COPY --chown=stump:stump --from=core-builder /app/stump ./app/stump
+COPY --from=core-builder /app/stump ./app/stump
 
 # copy the react build
 COPY --from=frontend /app/build ./app/client
 
+# Copy docker entrypoint
+# This will take care of starting the service daemon as a regular user, if desired
+COPY entrypoint.sh /entrypoint.sh
```

```suggestion
COPY scripts/release/entrypoint.sh /entrypoint.sh
```

It isn't ideal, but since this will be running from the project root, and not the `scripts/release` sub directory, I think we have to specify the entire relative path like that